### PR TITLE
Fixed Docker build-push-image

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

**Review by josvazg — APPROVED** [review]
LGTM

It would be good to link the tests we run for this manually as actions in this PR

**Comment by igor-karpukhin** [comment]
Good point. Here is a link to the run: https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/6824846574/job/18561555903

**Review by josvazg — APPROVED** [review]

**Comment by josvazg** [comment]
We will remove the comments on a follow up right?

**Review by helderjs — APPROVED** [review]
Looks good! Just don't forget to remove the commented code

File: .github/actions/certify-openshift-images/entrypoint.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -eou pipefail
 
 docker login -u mongodb+mongodb_atlas_kubernetes -p "${QUAY_PASSWORD}" quay.io
 
-DIGESTS=$(docker manifest inspect "quay.io/${REPOSITORY}:${VERSION}" | jq -r .manifests[].digest)
+DIGESTS=$(docker manifest inspect "quay.io/${REPOSITORY}:${VERSION}" | jq -r '.manifests[] | select(.platform.os!="unknown") | .digest')
```
Comment (helderjs) [inline .github/actions/certify-openshift-images/entrypoint.sh:7]: we are not supposed to have unknown images/digests. Should we avoid this happening instead?

Comment (igor-karpukhin) [reply .github/actions/certify-openshift-images/entrypoint.sh:7]: @helderjs We thought its because of `COPY` commands. cc: @s-urbaniak

Comment (s-urbaniak) [reply .github/actions/certify-openshift-images/entrypoint.sh:7]: yes, we have 3 layers here:
1. architecture-dependent: file system hierarchy and binaries
2. architecture-dependent: our operator binary
3. architecture-independent: trusted CA bundle
4. architecture-independent: license files

Comment (s-urbaniak) [reply .github/actions/certify-openshift-images/entrypoint.sh:7]: xref https://github.com/mongodb/mongodb-atlas-kubernetes/blob/ed2e4772b445bb790d0a7f5bda948249ca1dab03/fast.Dockerfile#L24-L26

**Comment by igor-karpukhin** [comment]
I'm merging this PR then. It's been tested with the pre-release registry

**Pull request merged**